suite/src/__tests__/fast/model-correctness.test.ts

```diff
@@ -40,15 +39,21 @@ describe('Model Integration Test', () => {
   })
 
   test('Create a ModelInstanceDocument on one node and read it from another', async () => {
+    const isIndexed = await TestUtils.waitForConditionOrTimeout(async () => {
```

Comment:
can we move this into the test harness setup rather than having it in a specific test body?

Reply:
Also might be nice to create a `waitForModelToBeIndexed` (or something like that) helper function in the utils directory.